aws/resource_aws_autoscaling_group.go

```diff
@@ -1493,3 +1527,75 @@ func flattenAutoScalingMixedInstancesPolicy(mixedInstancesPolicy *autoscaling.Mi
 
 	return []interface{}{m}
 }
+
```

Comment:
Is there ever a case when we'd want to set a timeout on these functions?

Reply:
Great question and yes, there might be! Quickly scanning the recent issues, I don't see any complaints about the similar change to the target group handling (that also does not have timeouts), but for now this is dependent on the AWS side entering some sort of failure state rather than staying in the adding/deleting state forever in those cases. 🤞